src/lib/context.py

```diff
@@ -139,7 +139,7 @@ def __init__(
         self.project_id_owner = project_id_owner
         self.dynatrace_api_key = dynatrace_api_key
         self.dynatrace_url = dynatrace_url
-        self.function_name = os.environ.get("FUNCTION_NAME", "Local")
+        self.function_name = os.environ.get("K_SERVICE", "Local")
```

Comment:
why you are changing this?

Reply:
I've changed it because in the newer runtime the ````FUNCTION_NAME```` variable is not available. https://cloud.google.com/functions/docs/configuring/env-var#newer_runtimes

Reply:
This is also a reason why I'd to add manually the 
````
GCP_PROJECT: '$GCP_PROJECT'
FUNCTION_REGION: '$GCP_REGION'
````
to environment variables list.